100-level-AsyncAPI-Intro/step5.md

```diff
@@ -0,0 +1,20 @@
+In this intergalactic journey, you explored AsyncAPI, a powerful specification for crafting asynchronous APIs. The first module discusses what AsyncAPI is and why to use AsyncAPI, exploring real-world AsyncAPI use cases.
+
+Building on the basics, module two transitions into understanding Event-driven Architecture (EDA), showcasing how AsyncAPI fuels the design of both EDA and microservices.
+
+The third module explores the AsyncAPI specification. You learn about the structure of AsyncAPI and its different components, such as info, channels, components, and tags. You also learn to define operations, messages, and schemas, and the request/reply pattern.
```

Comment:
WDYT? 
```suggestion
The third module explores the AsyncAPI specification. You learned about the structure of an AsyncAPI document and its different components, such as info, channels, operations, and messages. You also learned to implement the request/reply pattern.
```

Reply:
IMO: we try to use active voice while documenting, but i understand the changes requested.